src/components/Form/index.js

```diff
@@ -16,22 +16,28 @@ class Form extends React.Component {
     disabled: PropTypes.bool,
     onSubmit: PropTypes.func.isRequired,
     title: PropTypes.string,
-  };
+  }
 
   constructor(props) {
-    super(props);
+    super(props)
+    this.onSubmit = this.onSubmit.bind(this)
```

Comment:
Seems like we use arrow functions to handle this binding in a number of places. Could we do the same here and avoid the `constructor` all together?